src/renderer/hooks/useMapServerQuery.js

```diff
@@ -0,0 +1,22 @@
+// @ts-check
```

Comment:
@gmaclennan if we didn't create an API layer, we would have to type each query as shown here